client/src/components/categorical/occupancy.js

```diff
@@ -17,9 +16,7 @@ class Occupancy extends React.Component {
     const width = 100;
     const height = 11;
 
-    const categories = _.filter(schema.annotations.obs, {
-      name: colorAccessor
-    })[0].categories;
+    const categories = schema.annotations.obsByName[colorAccessor]?.categories;
```

Comment:
_.filter doesn't seem covered in the description of the PR, nor does `obsByName` - could you clarify here or there?

Reply:
done!

`obsByName` feature was added some time ago, so we didn't have to keep reprocessing the schema for cases like this.   So this was an example of unnecessary compute being done because I did not previously finish refactoring.